package.json

```diff
@@ -19,18 +19,20 @@
     "build": "bsb -make-world",
     "clean": "bsb -clean-world",
     "start": "bsb -make-world -w",
-    "pretest": "rimraf tests/main.bs.js && yarn build && sed -i '1s;^;require(\"@babel/polyfill\");\n' tests/main.bs.js",
-    "test": "browserify tests/main.bs.js | tape-run --browser chrome --render=faucet"
+    "pretest": "rimraf tests/main.bs.js && yarn build && prepend tests/main.bs.js 'require(\"abortcontroller-polyfill/dist/abortcontroller-polyfill-only\");require(\"@babel/polyfill\");\n'",
+    "test": "browserify tests/main.bs.js | tape-run --render=faucet"
```

Comment:
`faucet` is resulting in the following error:
```
$ browserify tests/main.bs.js | tape-run --render=faucet
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: spawn faucet ENOENT
    at _errnoException (util.js:992:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
    at onErrorNT (internal/child_process.js:372:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
error Command failed with exit code 1.
```
Maybe a different renderer? Not setting one at all displays fine for me, but maybe my terminal client works differently.